moto/redshift/models.py

```diff
@@ -941,6 +943,36 @@ def delete_tags(self, resource_name, tag_keys):
         resource = self._get_resource_from_arn(resource_name)
         resource.delete_tags(tag_keys)
 
+    @staticmethod
+    def _generate_random_alphanum():
```

Comment:
Let's move this into utilities.utils.py, there's a similar method there already - makes sense to have them in the same place.

Reply:
I removed it entirely and now using the existing one; I didn't see it at first, thanks for pointing it out.